commit 94e67b9225048fdcf47f623b87cf6a8d2a060b17
Author: Gilles Filippini <pini@debian.org>
Date:   Sat May 30 16:29:19 2020 +0200

    Store ACME_DIRECTORY into ca.conf
    
    The use case is when one wants to script an account email address change.
    Something like:
      for ca_conf in <acme.sh_conf_home>/ca/*/ca.conf; do
        eval $(grep '^ACME_DIRECTORY=' "$ca_conf")
        acme.sh --update-account --config_home <acme.sh_conf_home> --server "$ACME_DIRECTORY" --accountemail <new_email>
      done

diff --git a/acme.sh b/acme.sh
index 0e3110a..d32f895 100755
--- a/acme.sh
+++ b/acme.sh
@@ -3464,6 +3464,7 @@ _regAccount() {
     return 1
   fi
   _savecaconf "ACCOUNT_URL" "$_accUri"
+  _savecaconf "ACME_DIRECTORY" "$ACME_DIRECTORY"
   export ACCOUNT_URL="$_accUri"
 
   CA_KEY_HASH="$(__calcAccountKeyHash)"
